cmd/tracee-ebpf/internal/flags/flags-filter.go

```diff
@@ -7,6 +7,9 @@ import (
 	tracee "github.com/aquasecurity/tracee/pkg/ebpf"
 )
 
+// MaxBpfStrFilterSize value should match MAX_STR_FILTER_SIZE defined in BPF code
+const MaxBpfStrFilterSize = 16
```

Comment:
what's the unite of measure? I guess bytes. I suggest to add it to the var name.

Reply:
In this case, it's 16 bytes, but also 16 chars